### DISCUSSION TITLE
Postal has stopped sending mail.

### DISCUSSION THREAD

**Discussion by fernandoSynetic** [postalserver/postal#3099] [Help with using Postal]
Hi!

My problem is that postal is not sending mail to the recipients.
Emails are being queued, but remain in Pending status. All services are UP according to postal status.

If I stop Postal and run it again, everything works again for a while, a few hours. 
I deleted the records from the queued_messages table, but after a while it failed again.

The last entry in the log is:
worker_1    | E, [2024-08-22T16:00:13.085556 #9] ERROR -- #<Bunny::Session:0xca58 postal@127.0.0.1:5672, vhost=postal, addresses=[127.0.0.1:5672]>: Uncaught exception from consumer #<Bunny::Consumer:280880 @channel_id=1 @queue=deliver-jobs-main @consumer_tag=bunny-1724316843000-145459870088>: #<Bunny::ChannelAlreadyClosed: cannot use a closed channel! Channel id: 1, closed due to a server-reported channel error: PRECONDITION_FAILED - delivery acknowledgement on channel 1 timed out. Timeout value used: 1800000 ms. This timeout value can be configured, see consumers doc guide to learn more> @ /usr/local/bundle/gems/bunny-2.20.3/lib/bunny/channel.rb:2075:in `raise_if_no_longer_open!'

I appreciate any help. Thank you.

**Comment by willpower232** [comment]
You might want to upgrade to Postal v3 so that RabbitMQ is no longer used and any issues with it are removed from your setup.

**Comment by fernandoSynetic** [comment]
I updated to v3 and it has been working perfectly for weeks. Thank you.
